lib/smart_proxy_dynflow/plugin.rb

```diff
@@ -1,11 +1,14 @@
+require 'proxy/log'
+require 'proxy/pluggable'
+require 'proxy/plugin'
 class Proxy::Dynflow
   class Plugin < Proxy::Plugin
     http_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
     https_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
 
     settings_file "dynflow.yml"
-    default_settings :database => '/var/lib/foreman-proxy/dynflow/dynflow.sqlite'
     default_settings :console_auth => true
+    default_settings :core_url => 'http://localhost:8001'
```

Comment:
I know this is a work in progress, but had an initial thought, it'd be more convenient if the service would listen on a port marked already for HTTP so we don't need to touch SELinux.  These are the options:

```
http_port_t                    tcp      80, 81, 443, 488, 8008, 8009, 8443, 9000
```

I think 8008 is free on a proxy/capsule.